physics/clm_lake.f90

```diff
@@ -754,6 +756,11 @@ SUBROUTINE clm_lake_run( &
                   weasd(i)      = weasdi(i)
                   snowd(i)      = snodi(c)                  ! surface_snow_thickness_water_equivalent_over_ice
 
+
+                  if (.not. icy(i)) then
+                     flag_lakefreeze(i)=.true.
```

Comment:
@JiliDong-NOAA Jili, should it be **if(icy(i)) then** when the lake grid point is frozen?

Reply:
@JiliDong-NOAA I see now, that this will be true only for new ice points. I agree with this change. Thank you.